testing/README.md

```diff
@@ -5,3 +5,9 @@ Before running the script, create `<test_suite_name>_config.txt` for the corresp
 ```
 After running the commend, the output can be found in `./test/<test_suite_name>`.
 
+# fix_io.py manual
+Before running the script, add "vpr_args["write_vpr_constraints"] = os.path.join(temp_dir, os.path.basename(args.circuit_file)[:-2]+"_constraint.xml")" after line 557 of run_vtr_flow.py and then run get_intermediate_file.py.
+```sh
+./fix_io.py <test_suite_name> -output_path ./tests -input_path ./fix_io
```

Comment:
shouldn't this be `./generate_fix_io.py`?

Also shouldnt the input and output path be swapped? Which folder is being used as the output?